proxy/src/main/java/com/wavefront/agent/ChannelStringHandler.java

```diff
@@ -136,7 +164,12 @@ protected void channelRead0(ChannelHandlerContext ctx, String msg) throws Except
     }
     if (!points.isEmpty()) {
       ReportPoint point = points.get(0);
-      pointHandler.reportPoint(point, pointLine);
+      String errorMessage = verifyPoint(point);
+      if (errorMessage != "") {
```

Comment:
if you don't have intellij, let's fix that but you can't do this in java, you need equals() (intellij will flag that)